src/layout/BlogPostLayout4.astro

```diff
@@ -0,0 +1,92 @@
+---
+// General blog page layout
+import Button from "../component/Button.astro";
+import Table from "../component/Table2.astro";
```

Comment:
This is the only change from the original `BlogPostLayout.astro`, switching from `Table.astro` to `Table2.astro`